test/jdk/java/net/httpclient/lib/jdk/httpclient/test/lib/http2/Queue.java

```diff
@@ -58,14 +58,27 @@ public synchronized boolean isClosing() {
         return closing;
     }
 
+    public synchronized boolean isOpen() {
+        return !closed && !closing;
+    }
+
+    public synchronized boolean putIfOpen(T obj) {
+        if (!isOpen()) return false;
```

Comment:
Hello Daniel, should there be a `Objects.requireNonNull(obj)` just to match what the `put` currently does?